sys-power/powerstat/powerstat-0.02.11.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Laptop power measuring tool"
+HOMEPAGE="https://launchpad.net/ubuntu/+source/${PN}"
+SRC_URI="http://kernel.ubuntu.com/~cking/tarballs/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/gzip"
```

Comment:
gzip is already in `@system` https://gitweb.gentoo.org/repo/gentoo.git/tree/profiles/base/packages#n29 we don't need to list it here.